scripts/bundleSize.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Size limit for all bundles used by each service (K)
+# Keep these +/- 10K
+min=662
+max=686
+
+services=( "news" "persian" "igbo" "yoruba" "pidgin" )
```

Comment:
Are we going to list all 40 services eventually? Could we may be get the generated services and process them in a way to get the service names?
`ls *.js | grep -E -v 'vendor|main` is a good place to start. What are your thoughts on this?

Reply:
Yeah, nice idea :)

Reply:
Nice idea @sadick254, probably something to handle in a follow up issue.

Reply:
My bash is rubbish, anyone know how to do this? :P

Reply:
@dr3 what did you run to get that? should be able to use a regex to grab what is between `$/` and `-` then remove vendor and main

Reply:
Created https://github.com/bbc/simorgh/issues/2133